### DISCUSSION TITLE
Enable "Show previews of messages" by default

### DISCUSSION THREAD

**Discussion by MadLittleMods** [element-hq/element-meta#1917] [Product Feedback]
Enable the "Show previews of messages" setting in the room list by default. It was an instant value boost once I discovered it.

![](https://user-images.githubusercontent.com/558581/135164109-ba06737e-3fc3-43bb-ae96-3cc5086ad34d.png)

This was also mentioned in [my initial user experience doc](https://docs.google.com/document/d/1PkUlVyZFM1stDdXbrJXuHPnQ-1S1Dh5KiDaGlYhiTAU/edit#heading=h.b3zhlaeocgv2) where I wrote down all the sticky pain points when first using Element. I only discovered this setting myself after 7 months of daily Element exposure. Here is [another instance](https://matrix.to/#/!zzGPdcyFqxIOJEijoI:matrix.org/$99krL2A4KR0WckY2aIgEncxEUpqDJKroGqYXF2snplQ?via=matrix.org&via=lant.uk&via=raim.ist) of someone having to ask where to enable it even after knowing about it.

**Comment by bmisiak** [comment]
Thanks for filing this!

I agree previews should be on by default, at least for DMs with people. Back before the redesign I suggested this design goal:
![image](https://user-images.githubusercontent.com/2332213/135165716-06c6b349-599c-49ca-8362-e56a6429c235.png)

... as a way of clearly separating personal chats from big chat rooms (in addition to squircles ;))

**Comment by turt2live** [comment]
out of interest: were previews on for DMs by default? We have a product requirement somewhere that says previews should be on for DMs, but I'm not sure it was ever tested properly...

**Comment by Palid** [comment]
I think that if we were to change the defaults we should also iterate on the previews design, as currently everything is very cluttered together and could use a little bit more whitespace. If you compare it to the suggested redesign it's a night and day difference in readability. 😄

**Comment by MadLittleMods** [comment]
@Palid Can you link the redesign?

It doesn't feel strictly necessary to block this on a redesign if people already get value and doesn't stand out to me. Plus it's already a designed feature in the app. We can iterate that separately.

**Comment by Palid** [comment]
@MadLittleMods it's not as much as a redesign but some minor differences between implementation and design. I personally think it could be worth it to get two birds with one stone here, especially that we're changing the defaults.
